tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -93,10 +93,10 @@ var (
 		Egress:   true,
 	}
 	configVersion      = ""
-	ingressName        = "ingress"
+	ingressName        = "ingressgateway"
 	productPageTimeout = 60 * time.Second
 
-	rulesDir                 = "kube" // v1 rules directory by default
+	rulesDir                 = "policy"
```

Comment:
i *feel* like by splitting `policy` rules from `networking` you will need to update logic here for the files below ( `routeAllRule`, `routeReviewsVersionsRules`, `routeReviewsV3Rule`, `tcpDbRule`, `bookinfo-gateway`).

Were you able to run this locally successfully?

Reply:
Got it.